clj-refactor.el

```diff
@@ -1372,6 +1381,38 @@ returns (used.ns.lib1 used.ns.lib2)"
           (setq next-use-clause (1+ next-use-clause)))
         (nreverse (-flatten libs))))))
 
+(defun cljr--extract-refer-all-namespaces ()
+  "Returns a list of all the namespaces that are required with :refer :all"
+  (cljr--goto-ns)
+  (when (search-forward ":require" nil t)
+    (let (nses
+          (require-end (save-excursion
+                         (progn
+                           (paredit-backward-up)
+                           (paredit-forward)
+                           (point)))))
+      (if (not (save-excursion (re-search-forward ":refer\s:all" require-end t)))
+          (message "There is no :refer :all in the ns declaration.")
+        (while (save-excursion (re-search-forward ":refer\s:all" require-end t))
```

Comment:
re-search-forward both moves point and returns a truthy value when it succeeds, so this while loop can be simplified slightly.